consensus/state.go

```diff
@@ -1681,13 +1681,18 @@ func (cs *State) recordMetrics(height int64, block *types.Block) {
 	cs.metrics.MissingValidators.Set(float64(missingValidators))
 	cs.metrics.MissingValidatorsPower.Set(float64(missingValidatorsPower))
 
-	cs.metrics.ByzantineValidators.Set(float64(len(block.Evidence.Evidence)))
+	// byzantine validators power and count is only for consensus evidence i.e. duplicate vote
```

Comment:
can we mention this in https://docs.tendermint.com/master/tendermint-core/metrics.html

Reply:
It's already there lol:
```Number of validators who tried to double sign```